server/tests/middlewares/passportCallback.spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-console */
+import passportCallback from '<middlewares>/passport/passportCallback';
+import getProfile from '<fixtures>/passportCallback';
+import chai from 'chai';
+import chaiAsPromise from 'chai-as-promised';
+
+chai.use(chaiAsPromise);
+const { expect } = chai;
+
+const done = (err, obj) => obj;
+
+describe('Passportcallback function', () => {
+  it('should signup successfully with google profile', async () => {
+    let profile = getProfile('google', true);
+    let res = await passportCallback(null, null, profile, done);
+    expect(res.oauthId).to.be.equal(profile.id);
```

Comment:
I have told you to always put a space between the variable definitions and assertions. It will help readability